ui/src/components/EntryListItem/EntryListItem.tsx

```diff
@@ -146,17 +148,30 @@ export const EntryItem: React.FC<EntryProps> = ({entry, focusedEntryId, setFocus
             </div>}
             <div className={styles.endpointServiceContainer}>
                 <Summary method={entry.method} summary={entry.summary} updateQuery={updateQuery}/>
-                <div className={styles.service}>
+                <div className={styles.resolvedName}>
+                    <Queryable
+                        query={`src.name == "${entry.src.name}"`}
+                        updateQuery={updateQuery}
+                        displayIconOnMouseOver={true}
+                        style={{marginTop: "-4px", marginRight: "10px"}}
+                    >
+                        <span
+                            title="Source Name"
+                        >
+                            {entry.src.name ? entry.src.name : entry.src.ip}
```

Comment:
when entry.src.name is null, undefined  or empty string it will show IP which is ok but then clicking on it will add filter of 
src.name == <value> which is not what user clicked on.. 
same goes to the "dest.name" in line 172

Reply:
@gadotroee it can only be `""` but not `null`. Yes in that case the filter is `src.name == ""`. We have already a clickable UI element for the `src.ip` and `dst.ip` so no need to duplicate them. We inform the user that this field is `src.name`.